packages/govuk-frontend/src/govuk/components/footer/footer.yaml

```diff
@@ -204,12 +204,6 @@ examples:
             text: Ligula Nullam Ultricies
```

Comment:
Could you also hide 'with meta' since 'with meta links and meta content' cover the other meta examples?